docs/system-architecture.md

```diff
@@ -0,0 +1,21 @@
+# System Architecture
+
+This diagram shows the system architecture. [🔒 Make a copy of this Lucid template for your own application](https://lucid.app/lucidchart/8851888e-1292-4228-8fef-60a61c6b57e7/edit).
+
+![System architecture](https://lucid.app/publicSegments/view/e5a36152-200d-4d95-888e-4cdbdab80d1b/image.png)
```

Comment:
I like the instructions that come with updating the architecture diagram, however, would it be cleaner for this image to not include those so the architecture can be seen without distraction?

Reply:
VPC network shows a /16 cidr block, it should be [/20](https://github.com/navapbc/template-infra/blob/a875997ffc3192c288053ab5652123475a73b21e/infra/modules/network/main.tf#L4C27-L4C41).

Reply:
A point of clarification, the diagram shows clients come in port 443 > then port 80 between the IGW and ALB, is this accurate? My understanding is the client can ingress on port 80 / 443 but upon hitting the ALB on port 80 they will immediately redirected to 443 forcing a secure connection. Then they are forwarded on 443 to the target group. Once they hit the target group the request will be routed on port 8000 to ECS. I think either we should show 443 for that entire section or expand on what is happening in the public subnet to illustrate the change.

Reply:
One last thing, we mention the CIDR for the VPC, should we also list the subnet CIDRs for each subnet group or would that create to much clutter on the diagram?

Reply:
> I like the instructions that come with updating the architecture diagram, however, would it be cleaner for this image to not include those so the architecture can be seen without distraction?

Good point, I can create a separate tab called instructions

Reply:
@shawnvanderjagt ok updated the diagram with your feedback.

just one thing i didn't change, i kept things as HTTP on port 80, since HTTPS isn't yet implemented. @jamesbursa started that piece on https://github.com/navapbc/template-infra/pull/287

Reply:
Perfect. Thank you sir.